crates/ironrdp-pdu/src/rdp/server_license/server_license_request/tests.rs

```diff
@@ -272,9 +272,9 @@ fn from_buffer_correctly_parses_server_license_request() {
 
 #[test]
 fn from_buffer_correctly_parses_server_license_request_no_certificate() {
-    let server_certificate_header_buffer = vec![
+    let server_certificate_header_buffer = [
         0x03, 0x00, // blob type
-        0x00, 0x00, // blob len
```

Comment:
It’s probably fine to leave the comment here

Reply:
Added back a108bf64038dca4d3f9e455d38508d3b19c6f322